lib/src/main/res/values/strings.xml

```diff
@@ -27,41 +27,43 @@
 
     <!-- Social Buttons -->
-    <string name="com_auth0_lock_social_unknown_placeholder">LOG IN WITH %s</string>
-    <string name="com_auth0_lock_social_yahoo">LOG IN WITH YAHOO!</string>
-    <string name="com_auth0_lock_social_linkedin">LOG IN WITH LINKEDIN</string>
-    <string name="com_auth0_lock_social_google_oauth2">LOG IN WITH GOOGLE</string>
-    <string name="com_auth0_lock_social_twitter">LOG IN WITH TWITTER</string>
-    <string name="com_auth0_lock_social_facebook">LOG IN WITH FACEBOOK</string>
-    <string name="com_auth0_lock_social_amazon">LOG IN WITH AMAZON</string>
-    <string name="com_auth0_lock_social_aol">LOG IN WITH AOL</string>
-    <string name="com_auth0_lock_social_box">LOG IN WITH BOX</string>
-    <string name="com_auth0_lock_social_evernote">LOG IN WITH EVERNOTE</string>
-    <string name="com_auth0_lock_social_evernote_sandbox">LOG IN WITH EVERNOTE (SANDBOX)</string>
-    <string name="com_auth0_lock_social_exact">LOG IN WITH EXACT</string>
-    <string name="com_auth0_lock_social_github">LOG IN WITH GITHUB</string>
-    <string name="com_auth0_lock_social_instagram">LOG IN WITH INSTAGRAM</string>
-    <string name="com_auth0_lock_social_miicard">LOG IN WITH MIICARD</string>
-    <string name="com_auth0_lock_social_paypal">LOG IN WITH PAYPAL</string>
-    <string name="com_auth0_lock_social_salesforce">LOG IN WITH SALESFORCE</string>
-    <string name="com_auth0_lock_social_salesforce_community">LOG IN WITH SALESFORCE (COMMUNITY)</string>
-    <string name="com_auth0_lock_social_salesforce_sandbox">LOG IN WITH SALESFORCE (SANDBOX)</string>
-    <string name="com_auth0_lock_social_soundcloud">LOG IN WITH SOUNDCLOUD</string>
-    <string name="com_auth0_lock_social_windowslive">LOG IN WITH MICROSOFT</string>
-    <string name="com_auth0_lock_social_yammer">LOG IN WITH YAMMER</string>
-    <string name="com_auth0_lock_social_baidu">LOG IN WITH 百度</string>
-    <string name="com_auth0_lock_social_fitbit">LOG IN WITH FITBIT</string>
-    <string name="com_auth0_lock_social_planningcenter">LOG IN WITH PLANNING CENTER</string>
-    <string name="com_auth0_lock_social_renren">LOG IN WITH 人人</string>
-    <string name="com_auth0_lock_social_thecity">LOG IN WITH THE CITY</string>
-    <string name="com_auth0_lock_social_thecity_sandbox">LOG IN WITH THE CITY (SANDBOX)</string>
-    <string name="com_auth0_lock_social_thirtysevensignals">LOG IN WITH 37 SIGNALS</string>
-    <string name="com_auth0_lock_social_vkontakte">LOG IN WITH VKONTAKTE</string>
-    <string name="com_auth0_lock_social_weibo">LOG IN WITH 新浪微博</string>
-    <string name="com_auth0_lock_social_wordpress">LOG IN WITH WORDPRESS</string>
-    <string name="com_auth0_lock_social_yandex">LOG IN WITH YANDEX</string>
-    <string name="com_auth0_lock_social_shopify">LOG IN WITH SHOPIFY</string>
-    <string name="com_auth0_lock_social_dwolla">LOG IN WITH DWOLLA</string>
+    <string name="com_auth0_lock_social_log_in">LOG IN </string>
```

Comment:
This kind of i18n won't fly with right-to-left languages since you are assuming `Log in` will be first.